rhp/v3/rhp.go

```diff
@@ -5,10 +5,19 @@ import (
 	"time"
 
 	"go.sia.tech/core/consensus"
+	rhpv2 "go.sia.tech/core/rhp/v2"
 	"go.sia.tech/core/types"
 	"lukechampine.com/frand"
 )
 
+const (
+	blocksPerYear     = 365 * 144
+	registryEntrySize = 256
+
+	// SectorSize is the size of one sector in bytes.
+	SectorSize = rhpv2.SectorSize
```

Comment:
I resisted doing this in other places. I think it's best to stick with a single definition, so that there's no mixing and matching.

I suppose there's an argument to made that each RHP should define its own constants. However, RHPv3 isn't *really* it's own protocol -- it's an extension of RHPv2. For example, RHPv3 code makes use of the `rhpv2.HostSettings` type, and `hostd` has a top-level `rhp` package that is used for both v2 and v3.